webrender/src/device/gfx/device.rs

```diff
@@ -493,8 +493,8 @@ impl<B: hal::Backend> Device<B> {
             fbos: FastHashMap::default(),
             rbos: FastHashMap::default(),
             descriptor_pools,
-            descriptor_pools_global,
-            descriptor_pools_sampler,
+            descriptor_pools_global: Some(descriptor_pools_global),
```

Comment:
should we consolidate all the GPU-related things into a single struct that is stored in `Option` instead of having many smaller options?